src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/ViewModels/DataSetViewModels.cs

```diff
@@ -0,0 +1,113 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters.SystemJson;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using System.Text.Json.Serialization;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+
+/// <summary>
+/// Describes and provides high-level information about a data set.
```

Comment:
Perhaps would drop the `Describes and` part i.e. `Provides high-level information about a data set.`